Task_1/Task_1.cpp

```diff
@@ -1,6 +1,106 @@
 ﻿#include <iostream>
 
+using namespace std;
+
+//ЧИСЛО СУДЬБЫ. Цифры полной даты рождения суммируются, пока не получится одна цифра. Разработайте функцию и вычислите свое число судьбы)
+#define MIN_YEAR 0
+#define MAX_YEAR 2021
+#define MIN_MONTH 1
+#define MAX_MONTH 12
+
+int enter(int number, string str);
+bool isCheckData(int number, int min, int max);
+bool isCheckDay(int d, int m, int y);
+int numberOfFate(int d, int m, int y);
+
 int main()
 {
-    std::cout << "Hello World!\n";
+    int year = 0, month = 0, day = 0;
+
+    cout << "Enter date of birth: " << endl;
+
+    year = enter(year, "Year");
+    if (isCheckData(year, MIN_YEAR, MAX_YEAR))
+    {
+        cout << "There is no such date." << endl;
+
+        return -1;
+    }
+    else {}
+
+    month = enter(month, "Month");
+    if (isCheckData(month, MIN_MONTH, MAX_MONTH))
+    {
+        cout << "There is no such date." << endl;
+
+        return -1;
+    }
+    else{}
+
+    day = enter(day, "Day");
+    if (isCheckDay(day, month, year))
+    {
+        cout << "Number of fate: " << numberOfFate(day, month, year) << endl;
+    }
+    else 
+    {
+        cout << "There is no such date." << endl;
+
+        return -1;
+    }
+
+    return 0;
+}
+
+int enter(int number, string str)
+{
+    cout << str << ": ";
+    cin >> number;
+
+    return number;
+}
+
+bool isCheckData(int number, int min, int max)
+{
+    if (min == 1 and number == 0) {}
+    else if (number >= min and number <= max)
+    {
+        return false;
+    }
+    else {}
```

Comment:
It looks weird
Don't understand

Reply:
bool isCheckData(int number, int min, int max)
{
    if (number < min or number > max)
    {
        return true;
    }
    else 
    { 
        return false;
    }
}

Reply:
![image](https://user-images.githubusercontent.com/71211228/111917739-a7ccb080-8a92-11eb-8176-9922573c6251.png)